packages/marketplace/src/components/Grid.tsx

```diff
@@ -562,10 +562,11 @@ export default class Grid extends React.Component<
             })
             .map((card) => {
               // Clone the cards and update the prop to trigger re-render
-              const cardElement = React.cloneElement(card, {
+
+              return React.cloneElement(card, {
                 activeThemeKey: this.state.activeThemeKey,
+                key: `${card.props.item.title}-${card.props.item.user}`,
```

Comment:
Do we have or can we make a util function that generates the key based on the card data? Then we don't need to worry about the logic being in two different places (since it's in the card class as well)

Reply:
I don't think there is a similar function in the Card component? The key has never been present or generated except for this part.